tests/core/pyspec/eth2spec/test/altair/light_client/test_sync.py

```diff
@@ -381,6 +381,74 @@ def run_lc_sync_test_upgraded_store_with_legacy_data(spec, phases, state, fork):
     yield from finish_lc_sync_test(test)
 
 
+@with_light_client
+@spec_state_test_with_matching_config
+@with_presets([MINIMAL], reason="too slow")
+def test_light_client_sync_no_force_update(spec, state):
```

Comment:
Would recommend moving this up by a function (above `run_lc_sync_test_upgraded_store_with_legacy_data`), so that the `test_..._with_legacy_data` wrappers are located next to the `run_...` definition.